### DISCUSSION TITLE
Add func scaled() for all vector types (cale it by another vector)

### DISCUSSION THREAD

**Discussion by azur-wolve** [godotengine/godot-proposals#10856] [Scripting]
(not sure if this is **Scripting** or **Core** category)

something like that

```gdscript
func Vector3.scaled(scaler: Vector3) -> Vector3:
    var out : Vector3
    out.x = scaler * self.x
    out.y = scaler * self.y
    out.z = scaler * self.z
    return out

# usage example

my_vec = my_vec.scaled(scaler_vec)
```

if there's already a function that does that just tell, haven't found it

**Comment by tetrapod00** [comment]
I think you just want the `*` operator? https://docs.godotengine.org/en/stable/classes/class_vector3.html#class-vector3-operator-mul-vector3

**Reply by azur-wolve** [reply]
you're completely right >)

can just close this thread
